reactive/archive_auth_mirror.py

```diff
@@ -1,14 +1,18 @@
 from charmhelpers.core import hookenv
 
-from charms.reactive import when, when_not
+from charms.reactive import when, when_not, hook
 
 from charms.layer.nginx import configure_site
 from charms.archive_auth_mirror import gpg, reprepro, utils
 
 
+@hook('install')
+def install():
+    utils.install_resources()
```

Comment:
I agree with @sparkiegeek, sure other @hook('install') could be run before, but what we care about here is that other @when* handlers defined in this charm are not run before that.